Vagrantfile

```diff
@@ -1,12 +1,79 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+$script_subscription_setup = <<SCRIPT
+# Attach subscriptions, enable repos, install packages
+# Fix this so we only attach one sub :-)
+subscription-manager list --installed | grep -q \"Red Hat OpenShift Enterprise\" || subscription-manager list --matches=\"*OpenShift Enterprise*\" --available --all --pool-only | subscription-manager attach --file=-
+subscription-manager repos --enable=\"rhel-7-server-rpms\" --enable=\"rhel-7-server-extras-rpms\" --enable=\"rhel-7-server-ose-3.0-rpms\"
+rpm -qa | grep -q epel-release || rpm -ivh http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum -y install wget git net-tools bind-utils iptables-services bridge-utils gcc python-virtualenv docker ansible && yum update -y && yum clean all
+SCRIPT
+
+$script_guest_additions_install = <<SCRIPT
+# Install Guest Additions
+mkdir /mnt/cdrom && mount /dev/cdrom /mnt/cdrom
+# We are expecting this to fail, as there is no XWindows on the VM
+/mnt/cdrom/VBoxLinuxAdditions.run || true
+umount -f /mnt/cdrom && rm -rf /mnt/cdrom
+SCRIPT
+
+$script_configure_docker = <<SCRIPT
+# Configure Docker
+grep -q \"insecure-registry 172.30.0.0/16\" /etc/sysconfig/docker || echo \"INSECURE_REGISTRY='--selinux-enabled --insecure-registry 172.30.0.0/16'\" | tee --append /etc/sysconfig/docker > /dev/null
+
+# Configure ssh access
```

Comment:
This is actually nothing to do with docker config, it's to do with the ansible install